<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Loan Payoff</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableLoans">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Loan Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Principle</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Interest (APR)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Period</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="entryLoanName">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="entryLoanPrinciple">
          <property name="prefix">
           <string>$</string>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QDoubleSpinBox" name="entryLoanAPR">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="entryLoanPeriod">
          <property name="suffix">
           <string> months</string>
          </property>
          <property name="maximum">
           <number>1200</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonAddLoan">
          <property name="text">
           <string>Add Loan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeleteLoan">
          <property name="text">
           <string>Delete Selected</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Monthly Loan Payments</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Payment Method</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QDoubleSpinBox" name="entryMonthlyPayment">
          <property name="prefix">
           <string>$</string>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="entryMethodPrinciple">
          <property name="text">
           <string>&amp;Low-Principle First (Snowball)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="entryMethodInterest">
          <property name="text">
           <string>High-Interest First (A&amp;valanche)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QwtPlot" name="plotLoans"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAddLoan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addLoan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDeleteLoan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteLoan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entryMethodPrinciple</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectMethodPrinciple()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entryMethodInterest</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectMethodInterest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entryMonthlyPayment</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshPlot(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addLoan()</slot>
  <slot>deleteLoan()</slot>
  <slot>selectMethodPrinciple()</slot>
  <slot>selectMethodInterest()</slot>
  <slot>refreshPlot(double)</slot>
 </slots>
</ui>
